config/callbacks/default.yaml

```diff
@@ -32,6 +32,8 @@ model_summary:
 write_h5_callback:
   max_queue_size: 100
   max_concurrent_writers: 4
+  normalization_type: "sigmoid" # Make sure to use the appropriate normalization for your use case.
```

Comment:
Can you keep the defaults to no normalization and no compression?

Reply:
Right, yes indeed. That's best.